contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,196 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What is Angular?
+
+Angular, according to the [official website](https://angular.io/docs), is described as follows:
+
+>Angular is a platform that makes it easy to build applications with the web. Angular combines declarative templates, dependency injection, end to end tooling, and integrated best practices to solve development challenges. Angular empowers developers to build applications that live on the web, mobile, or the desktop.
+
+Let us understand some key terms starting with what <i>declarative templates</i> mean.
+Consider the example of creating a simple web application as shown below.
+
+![Example](First_example.png)
+
+The application displays the name and id of a person. The <i>view</i> component (i.e., the part users see) is defined in Angular using <i>templates</i>. An Angular template contains HTML code and <tooltip content="Angular allows developers to create their own HTML like tags to perform additional tasks">custom tags</tooltip>.
+Here is the template for the above example.
+  
+<iframe src="https://stackblitz.com/edit/angular-3qy4j2?embed=1&file=src/app/heroes/heroes.component.html" style="width:85%; height:380px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe> 
+
```

Comment:
You can remove `placeholder="name"` from the example without losing any value.

Reply:
Removing the ID can simplify the example further. It doesn't add value as it is no different from the name.